From e8202038afca9520c4731896e86934f2ef469da0 Mon Sep 17 00:00:00 2001
From: Paulo Fassina <1872247+pfassina@users.noreply.github.com>
Date: Sat, 15 Jun 2024 20:17:08 -0700
Subject: [PATCH] dmenu qalc

---
 config.def.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.def.h b/config.def.h
index 97f4786..f59cb37 100644
--- a/config.def.h
+++ b/config.def.h
@@ -102,6 +102,7 @@ static const Key keys[] = {
 	{ MODKEY,                       XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
 	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
+	{ MODKEY|ShiftMask,             XK_equal,  spawn,          {.v = dmenuqalc} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} },
 	{ MODKEY,                       XK_f,      setlayout,      {.v = &layouts[1]} },
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[2]} },
-- 
2.44.1

