From f2c4d99b671c92db11528bf26b1f6b099ac124ce Mon Sep 17 00:00:00 2001
From: Paulo Fassina <1872247+pfassina@users.noreply.github.com>
Date: Tue, 11 Jun 2024 22:33:17 -0700
Subject: [PATCH] alpha  adjustements

---
 config.def.h | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index 588ed39..e9aee8f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -30,9 +30,14 @@ static const char *colors[][3]      = {
   [SchemeInfoNorm] = { col_gray3, col_gray1, "#000000"  }, // infobar middle  unselected {text,background,not used but cannot be empty}
 };
 static const unsigned int alphas[][3]      = {
-    /*               fg      bg        border*/
-  [SchemeNorm] = { OPAQUE, baralpha, borderalpha },
-	[SchemeSel]  = { OPAQUE, baralpha, borderalpha },
+    /*                 fg      bg        border*/
+  [SchemeNorm]     = { OPAQUE, baralpha, borderalpha },
+	[SchemeSel]      = { OPAQUE, baralpha, borderalpha },
+	[SchemeStatus]   = { OPAQUE, baralpha, borderalpha },
+	[SchemeTagsSel]  = { OPAQUE, baralpha, borderalpha },
+	[SchemeTagsNorm] = { OPAQUE, baralpha, borderalpha },
+	[SchemeInfoSel]  = { OPAQUE, baralpha, borderalpha },
+	[SchemeInfoNorm] = { OPAQUE, baralpha, borderalpha },
 };
 
 /* tagging */
-- 
2.44.1

