From e4faa02762418b4051b2ce6bb413249e7467da69 Mon Sep 17 00:00:00 2001
From: Paulo Fassina <1872247+pfassina@users.noreply.github.com>
Date: Thu, 13 Jun 2024 22:12:14 -0700
Subject: [PATCH] dmenu rice

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 84223d0..ff0cfab 100644
--- a/config.def.h
+++ b/config.def.h
@@ -84,7 +84,7 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_accnt, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-c", "-l", "10", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_gray2, "-sf", col_accnt, "-nhb", col_gray1, "-nhf", col_gray4, "-shb", col_gray2, "-shf", col_accnt,  NULL };
 static const char *termcmd[]  = { "st", NULL };
 
 static const Key keys[] = {
-- 
2.44.1

